export_snowflake/__init__.py

```diff
@@ -522,26 +528,126 @@ def flush_records(stream: str,
     db_sync.delete_from_stage(stream, s3_key)
 
 
+def transfer_data_in_parallel(s3_csv_lists, config, o, file_format_type):
```

Comment:
what is source and destination here? s3 -> local or local -> snowflake? Reflect that in function name for clarity

Reply:
Sure thing, I will update the function names to clarify, same for the `download_file` method.